### PR TITLE
Release: v0.6.2

### PR DESCRIPTION
**Pull request by thomas-nguy**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Comment by thomas-nguy** [comment]
for reference, the diff can be seen there

https://github.com/crypto-org-chain/ethermint/pull/13

**Review by yihuang — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,18 @@
 
 ## Unreleased
 
+*November 19, 2021*
+
+## v0.6.2
```
Comment (tomtau) [inline CHANGELOG.md:7]: is that change in gRPC (`QueryAccountRequest`) breaking -- i.e. will the v0.6.1 binary still be able to use that query against v0.6.2 node?

Comment (thomas-nguy) [reply CHANGELOG.md:7]: I think it is ok. Ethermint does not provide any query command and cronos query commands do not use it

Comment (yihuang) [reply CHANGELOG.md:7]: https://github.com/crypto-org-chain/ethermint/commit/9f1478e3d35c538dce3c26971e3f5e8f550c7685#diff-324e7e7d0d1ab912bb60777c369af7304487e0a49727c3b9cf600ae9ef7865caL6
Is it this one? it looks like the only protobuf change.

Comment (tomtau) [reply CHANGELOG.md:7]: yeah, to my limited understanding, it's only adding a few fields, so most protobuf tooling should be fine (deserialisers in the old version would ignore it / put it in unknown fields: https://developers.google.com/protocol-buffers/docs/proto3#unknowns )

Comment (yihuang) [reply CHANGELOG.md:7]: The only api change is the `TraceTx ` and `TraceBlock`, they are not exposed through CLI I think.

**Pull request merged**